src/genie/libs/parser/nxos/show_fabricpath.py

```diff
@@ -0,0 +1,77 @@
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+
+# ==============================
+# Schema for 'show fabricpath isis adjacency'
+# ==============================
+class ShowFabricpathIsisAdjacencySchema(MetaParser):
+
+    ''' Schema for "show fabricpath isis adjacency" '''
+
+# These are the key-value pairs to add to the parsed dictionary
+
+    schema = {
+        Any():
+            {'adj-hold-time-out': str,
+            'adj-intf-name-out': str,
+            'adj-sys-name-out': str,
+             'adj-state-out':str}
+    }
+
+# Python (this imports the Python re module for RegEx)
+import re
```

Comment:
Please move this to top of file.

Reply:
Please check commit: 4b4bfa2c51ea119eeef830de97bb979a3c74c800